app/controllers/shipit/api/base_controller.rb

```diff
@@ -0,0 +1,68 @@
+module Shipit
+  module Api
+    class BaseController < ActionController::Base
+      include Shipit::Engine.routes.url_helpers
```

Comment:
Indeed. It's already on master, I added it a while ago trying to debug something. I'll remove it.